src/zenml/integrations/pytorch/materializers/pytorch_materializer.py

```diff
@@ -16,7 +16,7 @@
 from typing import Any, Type, Union
 
 import torch
-from torch.nn import Module  # type: ignore[attr-defined]
+from torch.nn import Module
```

Comment:
mypy complains because this ignore comment was removed

Reply:
Fixed in the latest commits